test/snapshots/no-named-default.js.md

```diff
@@ -0,0 +1,497 @@
+# Snapshot report for `test/no-named-default.js`
+
+The actual snapshot is saved in `no-named-default.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## invalid(1): import {default as named} from "foo";
+
+> Input
+
+    `␊
+      1 | import {default as named} from "foo";␊
+    `
+
+> Output
+
+    `␊
+      1 | import named   from "foo";␊
+    `
+
+> Error 1/1
+
+    `␊
+    > 1 | import {default as named} from "foo";␊
+        |         ^^^^^^^^^^^^^^^^ Prefer using the default import over named import.␊
+    `
+
+## invalid(2): import {default as named,} from "foo";
+
+> Input
+
+    `␊
+      1 | import {default as named,} from "foo";␊
+    `
+
+> Output
+
+    `␊
+      1 | import named   from "foo";␊
+    `
+
+> Error 1/1
+
+    `␊
+    > 1 | import {default as named,} from "foo";␊
+        |         ^^^^^^^^^^^^^^^^ Prefer using the default import over named import.␊
+    `
+
+## invalid(3): import {default as named, bar} from "foo";
+
+> Input
+
+    `␊
+      1 | import {default as named, bar} from "foo";␊
+    `
+
+> Output
+
+    `␊
+      1 | import named,  { bar} from "foo";␊
+    `
+
+> Error 1/1
+
+    `␊
+    > 1 | import {default as named, bar} from "foo";␊
+        |         ^^^^^^^^^^^^^^^^ Prefer using the default import over named import.␊
+    `
+
+## invalid(4): import {default as named, bar,} from "foo";
+
+> Input
+
+    `␊
+      1 | import {default as named, bar,} from "foo";␊
+    `
+
+> Output
+
+    `␊
+      1 | import named,  { bar,} from "foo";␊
+    `
+
+> Error 1/1
+
+    `␊
+    > 1 | import {default as named, bar,} from "foo";␊
+        |         ^^^^^^^^^^^^^^^^ Prefer using the default import over named import.␊
+    `
+
+## invalid(5): import defaultExport, {default as named} from "foo";
+
+> Input
+
+    `␊
+      1 | import defaultExport, {default as named} from "foo";␊
+    `
+
+> Output
+
+    `␊
+      1 | import named from "foo";␊
+      2 | import defaultExport from "foo";␊
```

Comment:
I choose to duplicate `import` instead of 

```js
import defaultExport from "foo";
const named = defaultExport;
```

As mentioned in https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2087#issuecomment-1543530438 and https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2087#issuecomment-1543546434, because theoretically it can be a `let`, live binding is safer.

Anyway, it's just an edge case.